R/recall_data.R

```diff
@@ -191,28 +200,41 @@ recall_details <- function(recall_number, limit = 25) {
             call. = FALSE)
     }
 
-    compiled_df <- vector(mode = "list", length = length(recall_number))
+
     i <- 1
     for (single_number in recall_number) {
 
+        #initialize dataframe
+        if (i == 1){
```

Comment:
Better practice to compile a list of data and then convert the list vectors (or list of lists, or list of dataframe) to a dataframe at the at end. Rbind isn't a great operation as I believe it copies each column vector every time it is called. I realize that this may be because `clean_vrd_api()` returns a dataframe. So if creating a list of dataframe and then combining ends up being difficult can leave as is. 

Also (whether the above note is resolved or not), should probably avoid having a different piece of code for `i=1`. Two options to do this are to either 1) make a call to your function upfront with no data which could return an empty df with the required columns or 2) set an empty df with the required columns at the top.